plugins/Sid/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+INCLUDE(BuildPlugin)
+
+INCLUDE_DIRECTORIES(resid)
+
+BUILD_PLUGIN(sid
+	SidInstrument.cpp
```

Comment:
The intent was to use `Sid.cpp` and name Class=Plugin like we've done for the other submodules, but I was afraid that Windows machines would confuse `sid.h` and `Sid.h`, so I kept the `Instrument` prefix.